fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -296,6 +309,87 @@ public DataApiRequest(
     }
 
     /**
+     * To check whether dateTime column request is first one in the sort list or not.
+     *
+     * @param sortColumns  map of columns and its direction
+     *
+     * @return True if dateTime column is first one in the sort list. False otherwise
+     */
+    protected Boolean isDateTimeFirstSortField(Map<String, SortDirection> sortColumns) {
```

Comment:
Please make it take a `List` in a follow-on PR. It really doesn't feel like we're doing Map operations for most of this logic, it feels much more list-y (and the JavaDoc reflects that as well). It can stay a LHM for now, but we should correct the mis-match.